db/operations.go

```diff
@@ -151,8 +176,28 @@ func formatValue(tableName, columnName, value string, valueType reflect.Type) (s
 			v := time.Unix(int64(i), 0).Format(time.RFC3339)
 			return fmt.Sprintf("'%s'", v), nil
 		}
-		return "", fmt.Errorf("unsupported type %s for column %s in table %s", valueType, columnName, tableName)
+		return "", fmt.Errorf("unsupported struct type %s", valueType)
 	default:
-		return "", fmt.Errorf("unsupported type %s for column %s in table %s", valueType, columnName, tableName)
+		return "", fmt.Errorf("unsupported type %s", valueType)
+	}
+}
+
+func EscapeString(valueToEscape string, escapeType string) (string, error) {
```

Comment:
Should be private, not used outside module. 

Moreover, I'm wondering about the performance impact we have here. Ideal world, we would "scan" the string and replace all characters. At least `strings.ReplaceAll` is a no-op more or less if there is no occurrences of the said characters. If there is multiple replacements, it would be another story.

I could try to run some test before/after this is merged to see the impact.